test/sslapitest.c

```diff
@@ -4927,6 +4955,9 @@ static int test_export_key_mat(int tst)
     if (tst == 1)
         return 1;
 #endif
+    /* TODO(3.0): No TLSv1.0/TLSv1.1 support with FIPS at the moment */
```

Comment:
I thought we agreed to not supporting v1.X in FIPS MODE?

Reply:
I removed the comment